docs/src/main/sphinx/optimizer/pushdown.rst

```diff
@@ -271,3 +271,98 @@ FETCH FIRST N ROWS``.
 
 Implementation and support is connector-specific since different data sources
 support different SQL syntax and processing.
+
+Below are showcased two similar queries for demonstrating how the Trino
```

Comment:
Find two similar ... in the following sections. First, an example of a Top-N query...